### PR TITLE
added faq

### PR DESCRIPTION
**Pull request by Akshay-Srivatsan**

### CODE REVIEW

**Review by jreinstra — APPROVED** [review]
lgtm

**Comment by jreinstra** [comment]
maybe let jason take a quick look

**Review by jasonscharff — CHANGES_REQUESTED** [review]
I'm looking at the website and it seems way too long now. We made some changes a while ago to shrink this and I think this creates the problem again.

There are a few possible solutions I can think of:
1. Reduce the FAQ to just be about travel reimbursement + high schoolers. Disadvantage is it looks stupid to have a FAQ with two things, but that's the catalyst. Advantage is it's easy to implement.
2. Make the questions collapsed by default and when you click it on a card comes out. Ashley might have ideas on this. Disadvantage is its harder to implement. Advantage is its nicer for the user.

File: index.html [hunk 1]
```diff
@@ -86,7 +86,63 @@ <h2>Travel</h2>
...
+                </div>
+            </div>
+            <div class="row">
+                <div class="six columns">
+                    <h3>Is there a code of conduct?</h2>
+                    <p>Yes, there is. Nothing too out of the ordinary though. Generally, be nice to others. If you want the full guide, we are using the MLH code of conduct which can be found <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">here</a>.</p>
+                </div>
+                <div class="six columns">
+                    <h3>I have other questions.</h2>
+                    <p>You can contact us at <a href="mailto:hello@menlohacks.com">hello@menlohacks.com</a>, and we will do our best to answer your questions.</p>
```
Comment (jasonscharff) [inline index.html]: we don't want to email hello@menlohacks.com

**Review by jasonscharff — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Akshay-Srivatsan) [reply index.html]: maybe I took a bit too much inspiration from MenloHacks...

**Comment by Akshay-Srivatsan** [comment]
I think a basic collapsing view wouldn't be too hard to implement--I can create a prototype and add it to this PR

**Comment by jasonscharff** [comment]
sweet.

File: index.html (hunk 1 above)
Comment (jasonscharff) [reply index.html]: lol don't worry––this is what code review is for.

**Review by jasonscharff — APPROVED** [review]
good enough to 🚢

**Pull request merged**